scripts/core/EXP-COMMAND-BUFFER.rst

```diff
@@ -173,7 +173,14 @@ ${X}_DEVICE_INFO_COMMAND_BUFFER_UPDATE_SUPPORT_EXP query.
 Updating kernel commands is done by passing the new kernel configuration
 to ${x}CommandBufferUpdateKernelLaunchExp along with the command handle of
 the kernel command to update. Configurations that can be changed are the
-parameters to the kernel and the execution ND-Range.
+kernel handle, the parameters to the kernel and the execution ND-Range.
+
+Kernel handles that might be used to update the kernel of a command, need
```

Comment:
Is the original `hkernel` kernel an implicit alternative. i.e. if you updated back from the original kernel to an alternative, then back to original, would that still work? Might be worth adding a sentence to the spec on as well as a test

Reply:
Yes, should still work. Will update the spec to make this behavior more clear :+1: